source/routes/update/user.js

```diff
@@ -0,0 +1,44 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
+const bcrypt = require('bcrypt');
+const authenticate = require(_base + 'middleware/authenticate');
+
+// Update a user's data
+// Request json must be in the form:
+// {
+// "username": "User's username (primary index)",
+// "updateField": {"customFieldToUpdate": "customDataToUpdate"}
+// }
+module.exports = {
+  '/update/user': {
+    methods: ['post'],
+    middleware: [authenticate],
+    fn: function(req, res, next) {
+      if (!req.body.updateField) {
+        throw new Error('MISSING UPDATE DATA');
+      }
+      db.get(req.body.username)
+        .then((response) => {

```

Comment:
Codacy has a fix for the issue: [Expected indentation of 10 spaces but found 8.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7405452)
```suggestion
          .then((response) => {
```